npm-shrinkwrap.json

```diff
@@ -1,1911 +1,2131 @@
 {
```

Comment:
The shrinkwrap doesnt seem to working. getting an error when doing 'npm install'